src/components/nav-icon-item/styles.ts

```diff
@@ -0,0 +1,61 @@
+import { styled } from '@linaria/react'
+import { AnchorHTMLAttributes, ButtonHTMLAttributes } from 'react'
+
+type CommonNavIconItemProps = {
+  /**
+   * NOTE: data attribute to indicate the state of the nav item
+   * by default the styles is already applied with pseudo-class selector
+   * but to make it more flexible, consumer could use this attribute to
+   * change the styles
+   */
+  'data-state'?: 'focus' | 'hover' | 'active'
+}
+
+const baseStyles = `
+  display: inline-flex;
+  padding: var(--spacing-2, 8px);
+  justify-content: center;
+  align-items: center;
+  gap: var(--spacing-none, 0px);
+  border-radius: var(--corner-default, 0.25rem);
+  background: var(--white, #ffffff);
+  border: none;
+  outline: none;
+
+  /* change children color by default */
+  > * {
+    color: var(--neutral-400, #798da1) !important;
+  }
+
+  &:focus, &[data-state='focus'] {
+    border-radius: var(--corner-default, 0.25rem);
+    background: var(--white, #ffffff);
+    box-shadow:
+      0px 0px 0px 1px #fff,
+      0px 0px 0px 4px var(--icon-button_primary-hover, #7e9bfa);
+  }
+
+  &:hover, &[data-state='hover'] {
+    cursor: pointer;
+    border-radius: var(--corner-default, 0.25rem);
+    background: var(--neutral-050, #f2f4f6);
+  }
+
+  &:active, &[data-state='active'] {
+    > * {
+      color: var(--intent-primary, #4e56ea) !important;
```

Comment:
**issue:** `--intent-primary` is not, I believe, a Design System token; rather, I think it's an old token from Elements v4 (unless I'm blind and just can't see this in the DS). The DS tokens are outlined in https://www.figma.com/design/6CaivqdlTX0UkFYJkpBKDu/Reapit-DS?node-id=1800-11798&m=dev. Can you please ensure all CSS variables being used here are listed in the linked Figma file 🙏

Reply:
> think it's an old token from Elements v4

right, I'll check it again to ensure it's using the latest design system